src/cache/client/npm/vuln-regex-detector-client.js

```diff
@@ -418,7 +440,7 @@ function kvPersistentFname (config, key) {
 	 * Using a hash might give us false reports on collisions, but this is
 	 * exceedingly unlikely in typical use cases (a few hundred regexes tops). */
 	const hash = crypto.createHash('md5').update(key).digest('hex');
-	const fname = path.join(config.cache.persistentDir, `${hash}.json`);
+	const fname = path.join(config.cache.persistentDir, `${hash}-v2.json`);
```

Comment:
Do you think we should the number "2" a global `CACHE_VERSION` variable? Then a comment on that variable can indicate history better.